src/build.py

```diff
@@ -267,12 +267,13 @@ def JavaBinDir():
 NPROC = multiprocessing.cpu_count()
 
 
-if IsMac():
+if IsMac() or True:
   # Experimental temp fix for crbug.com/829034 stdout write sometimes fails
-  from fcntl import fcntl, F_GETFL, F_SETFL
+  from fcntl import fcntl, F_GETFL
   fd = sys.stdout.fileno()
   flags = fcntl(fd, F_GETFL)
-  fcntl(fd, F_SETFL, flags & ~os.O_NONBLOCK)
+  print 'Flags: %s, nonblock: %s' % (hex(flags), flags & os.O_NONBLOCK)
```

Comment:
- You used Python 3 style `print` in `testing.py`... did you mean to do it here too?
- Not sure what that exactly means, but code in the left side is `flag & ~os.O_NONBLOCK` and this is `flags & ~os.O_NENBLOCK`, with missing `~`. Is this intentional?

Reply:
* No I just converted that one file because with the print function I could to a search and replace for sys.stdout.write and put print instead. But I didn't change the rest of the files. If this PR doesn't help I'll probably just revert it. Otherwise I'll go in and clean it up
* Previously the code was clearing the nonblock flag, and now the code is just printing the value